contrib/emicklei/go-restful/restful.go

```diff
@@ -0,0 +1,37 @@
+// Package restful provides functions to trace the emicklei/go-restful package (https://github.com/emicklei/go-restful).
+package restful
+
+import (
+	"strconv"
+
+	"github.com/emicklei/go-restful"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// Trace is a filter that will trace incoming request
```

Comment:
Since this is imported as `restfultrace`, I think it would be nicer to call it `Filter`, then it would be used as `restfultrace.Filter` which IMHO is better.
```go
// Filter is a FilterFunction which traces a request.
func Filter(req *restful.Request, resp *restful.Response, chain *restful.FilterChain) {
```

Reply:
ah good point, will change it to `Filter`